src/sentry/api/serializers/models/environment.py

```diff
@@ -22,6 +23,15 @@ def serialize(self, obj, attrs, user):
         }
 
 
+@register(EnvironmentProject)
+class EnvironmentProjectSerializer(Serializer):
+    def serialize(self, obj, attrs, user):
+        return {
+            'name': obj.environment.name,
+            'is_hidden': obj.is_hidden is True,
+        }
```

Comment:
It's sort of weird this return type is different from `EnvironmentSerializer` which is used for the non-details endpoint…?